preupg/cli.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function, unicode_literals
-#import argparse
 import optparse
```

Comment:
I'm not sure if the argparse is backwards compatible. If yes, then why not. We may try it one time. However currently, neither RHEL 5 nor RHEL 6 have argparse available so this RFE is for quite a distant future.